docs/public/content/guide/routing.md

```diff
@@ -16,13 +16,13 @@ In this section, we'll cover the 3 kinds of routes you can find in an __elm-spa_
 
 ## The homepage
 
-The `src/Pages/Home_.elm` file is a reserved page that handles requests to `/`. The easiest way to make a new homepage is with the [`add` command](/guide/cli#adding-a-homepage) covered in the CLI section:
+The `src/Pages/Home_.elm` file is a __dynamic route__ that's reserved and handles requests to `/`. The easiest way to make a new homepage is with the [`add` command](/guide/cli#adding-a-homepage) covered in the CLI section:
```

Comment:
The `Home_.elm` file isn't a dynamic route, despite the trailing `_`. 

Maybe the first sentence should be more like this:
> `Home_.elm` is a reserved filename that handles requests to `/`.

If you could add something like that instead, this change looks good to me.